Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM python:3.8-slim-buster
-WORKDIR /usr/src/app
-COPY requirements.txt ./
-RUN pip3 install --upgrade pip
-RUN pip3 install --no-cache-dir -r requirements.txt
-COPY . .
+FROM python:3.9-slim
+
+RUN useradd -ms /bin/bash kvuser
+USER kvuser
+WORKDIR /home/kvuser
+
+ENV PATH="/home/kvuser/.local/bin:${PATH}"
+
+COPY --chown=kvuser:kvuser requirements.txt requirements.txt
+RUN python3 -m pip install --upgrade pip
```

Comment:
This uses only extra space in the image and in the future may lead to breakage
```suggestion
```

Reply:
There was removed string RUN python3 -m pip install --upgrade pip
recreated and uploaded to the ECR new image